cf.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "------ Create database tables ------"
+
+python manage.py migrate --noinput
+python manage.py load_agency_contacts
```

Comment:
@khandelwal the buildpack itself runs collectstatic https://github.com/cloudfoundry/python-buildpack/blob/master/bin/compile#L191. You don't have to run it here